### PR TITLE
Remove examples from the schema file

### PR DESCRIPTION
**Pull request by jbkoh**
Previously, example entities are hardcoded inside Brick.ttl because generate_brick.py does the generation and testing inference together. I separate the inference testing functionality into ``tests/test_inference.py``. And now we can add more testing functionality into the directory too.

### CODE REVIEW

File: generate_brick.py [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
 
-BRICK = Namespace("https://brickschema.org/schema/1.1.0/Brick#")
-TAG = Namespace("https://brickschema.org/schema/1.1.0/BrickTag#")
-BLDG = Namespace("https://brickschema.org/schema/1.1.0/ExampleBuilding#")
-OWL = Namespace("http://www.w3.org/2002/07/owl#")
-RDF = Namespace("http://www.w3.org/1999/02/22-rdf-syntax-ns#")
-RDFS = Namespace("http://www.w3.org/2000/01/rdf-schema#")
+BRICK = Namespace("https://brickschema.org/schema/{0}/Brick#".format(BRICK_VERSION))
+TAG = Namespace("https://brickschema.org/schema/{0}/BrickTag#".format(BRICK_VERSION))
+BLDG = Namespace("https://brickschema.org/schema/{0}/ExampleBuilding#".format(BRICK_VERSION))
```
Comment (gtfierro) [inline generate_brick.py]: You can remove the `BLDG` namespace

**Comment by gtfierro** [comment]
Thanks for pulling the test code out of there. Next steps should eventually be to wrap the tests in an actual Python test framework and hook that up to Travis CI

File: generate_brick.py (hunk 1 above)
Comment (jbkoh) [reply generate_brick.py]: done.

**Comment by jbkoh** [comment]
right, I am writing another testing script. Let's collect them inside ``tests/`` dir and then configure a CI framework.

**Comment by gtfierro** [comment]
:shipit:

**Pull request merged**